app/script/entity/Conversation.js

```diff
@@ -43,7 +43,7 @@ z.entity.Conversation = class Conversation {
     const inputStorageKey = `${z.storage.StorageKey.CONVERSATION.INPUT}|${this.id}`;
     this.input = ko.observable(this._getStorageInputData(inputStorageKey));
     this.input.subscribe(({mentions, text}) => {
-      return z.util.StorageUtil.setValue(inputStorageKey, {mentions, text: text.trim()});
+      return z.util.StorageUtil.setValue(inputStorageKey, {mentions, text: text.trimEnd()});
```

Comment:
If we trim the whitespace in the beginning, the loaded mentions will be in the wrong position.

Reply:
:+1: for the explanation here.